src/test/java/com/houarizegai/calculator/ui/CalculatorUI_calculate_755b65b191_Test.java

```diff
@@ -1,60 +1,11 @@
-// ********RoostGPT********
-/*
-Test generated by RoostGPT for test demoTestGitlab using AI Type Open AI and AI Model gpt-4
-
-Here are some test scenarios for the function calculate:
-
-1. Test Scenario: Addition of two positive numbers
-   - Input: firstNumber = 4, secondNumber = 5, operator = '+'
-   - Expected Output: 9.0
-
-2. Test Scenario: Subtraction of two positive numbers where the first number is greater than the second
-   - Input: firstNumber = 10, secondNumber = 5, operator = '-'
-   - Expected Output: 5.0
-
-3. Test Scenario: Subtraction of two positive numbers where the second number is greater than the first
-   - Input: firstNumber = 2, secondNumber = 5, operator = '-'
-   - Expected Output: -3.0
-
-4. Test Scenario: Multiplication of two positive numbers
-   - Input: firstNumber = 3, secondNumber = 5, operator = '*'
-   - Expected Output: 15.0
-
-5. Test Scenario: Division of two positive numbers where the first number is greater than the second
-   - Input: firstNumber = 10, secondNumber = 5, operator = '/'
-   - Expected Output: 2.0
-
-6. Test Scenario: Division of two positive numbers where the second number is greater than the first
-   - Input: firstNumber = 5, secondNumber = 10, operator = '/'
-   - Expected Output: 0.5
-
-7. Test Scenario: Division by zero
-   - Input: firstNumber = 5, secondNumber = 0, operator = '/'
-   - Expected Output: Infinity (or throw an exception)
-
-8. Test Scenario: Modulus of two positive numbers
-   - Input: firstNumber = 10, secondNumber = 3, operator = '%'
-   - Expected Output: 1.0
-
-9. Test Scenario: Power of two positive numbers
-   - Input: firstNumber = 2, secondNumber = 3, operator = '^'
-   - Expected Output: 8.0
-
-10. Test Scenario: Invalid operator
-    - Input: firstNumber = 2, secondNumber = 3, operator = 'a'
-    - Expected Output: 3.0
-*/
-
-// ********RoostGPT********
 package com.houarizegai.calculator.ui;
 
 import com.houarizegai.calculator.CalculatorUI;
```

Comment:
this import statement should either be removed or can be improved as import com.houarizegai.calculator.ui.CalculatorUI